README.md

```diff
@@ -291,18 +292,25 @@ If you do want to re-enable access from your development environment, re-run the
 azd provision
 ```
 
-## Costs
+## Guidance
 
-Pricing may vary per region and usage. Exact costs cannot be estimated.
-You may try the [Azure pricing calculator](https://azure.microsoft.com/pricing/calculator/) for the resources below:
+### Region Availability
 
-* Azure Container Apps: Pay-as-you-go tier. Costs are based on vCPU and memory used. [Pricing](https://azure.microsoft.com/pricing/details/container-apps/)
-* Azure OpenAI: Standard tier, GPT, and Ada models. Pricing per 1K tokens used, and at least 1K tokens are used per question. [Pricing](https://azure.microsoft.com/pricing/details/cognitive-services/openai-service/)
-* Azure Monitor: Pay-as-you-go tier. Costs based on data ingested. [Pricing](https://azure.microsoft.com/pricing/details/monitor/)
+This template uses [MODEL 1] and [MODEL 2] which may not be available in all Azure regions. Check for [up-to-date region availability](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#standard-deployment-model-availability) and select a region during deployment accordingly
+  * We recommend using [SUGGESTED REGION]
 
-## Security Guidelines
+### Costs
 
-This template uses [Managed Identity](https://learn.microsoft.com/entra/identity/managed-identities-azure-resources/overview) built in to eliminate the need for developers to manage these credentials. Applications can use managed identities to obtain Microsoft Entra tokens without having to manage any credentials. We also use Key Vault, specifically for Bing Search, since Managed Identity is currently not implemented for it. Additionally, we have added a [GitHub Action tool](https://github.com/microsoft/security-devops-action) that scans the infrastructure-as-code files and generates a report containing any detected issues. To ensure best practices in your repo we recommend anyone creating solutions based on our templates ensure that the [Github secret scanning](https://docs.github.com/code-security/secret-scanning/about-secret-scanning) setting is enabled in your repos.
+You can estimate the cost of this project's architecture with [Azure's pricing calculator](https://azure.microsoft.com/pricing/calculator/)
+
+* [Azure Product] - [plan type] [link to pricing for product](https://azure.microsoft.com/pricing/)
```

Comment:
@marlenezw please update this generic section with actual product pricing information before merge